src/components/journeys/JourneyInstancesDataTable/getStaticColumns.tsx

```diff
@@ -0,0 +1,72 @@
+import dayjs from 'dayjs';
+import { GridColDef } from '@mui/x-data-grid-pro';
+
+import PersonHoverCard from 'components/PersonHoverCard';
+import ZetkinPerson from 'components/ZetkinPerson';
+import ZetkinRelativeTime from 'components/ZetkinRelativeTime';
+import { ZetkinJourney, ZetkinPerson as ZetkinPersonType } from 'types/zetkin';
+
+// Name concatenation
+const getPeopleString = (people: ZetkinPersonType[]) =>
```

Comment:
Might be nice to make a new util in `src/utils.ts` which has a little util for building the name from first and last, and this one. Just an idea.